docs/howto/deployment/asgi/uvicorn.txt

```diff
@@ -12,7 +12,7 @@ Installing Uvicorn
 
 You can install Uvicorn with ``pip``::
 
-    python -m pip install uvicorn
+    python -m pip install uvicorn gunicorn
```

Comment:
should this be updated to use the standard edition of uvicorn, rather than the specialist pure python version:

```suggestion
    python -m pip install uvicorn[standard] gunicorn
```

Reply:
Hey @graingert. That's not very pretty; it seems a bit backwards to make the default install the specialist one, but I'm not up to date on what's the advice there? (I guess we need to go with whatever is recommended.)

Reply:
sadly it's the way pypa extras work - they can only be additive and not subtractive

Reply:
OK, thanks. I'll glance over the unicorn docs just to refresh my memory there and we'll go with this. 🥇

Reply:
Hmmm. The quickstart doesn't have this. https://www.uvicorn.org/ Just says `pip install uvicorn`

Reply:
https://github.com/encode/uvicorn#quickstart

Reply:
@carltongibson looks like the readme.md and the docs/index.md got out of sync

https://github.com/encode/uvicorn/pull/784/files

Reply:
OK, thanks. I'll look again later on and read through it. (Sorry, it's been a while since I've played with unicorn so I'm a bit out of the loop.)